src/main/java/org/fundacionjala/coding/mmendieta/Main.java

```diff
@@ -0,0 +1,15 @@
+package org.fundacionjala.coding.mmendieta;
+
+public final class Main {
```

Comment:
Since now you will use unit tests, this class will no longer necessary.
Please remove it!.